content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -282,22 +282,31 @@ Full <abbr>parameters</abbr> (key and value) are listed separately under
 
 ## Structure of the cache directory
 
+The DVC cache is a
+[content-addressable storage](https://en.wikipedia.org/wiki/Content-addressable_storage)
+(CAS), which adds a layer of indirection between code and data. It's hidden from
+the user (and from Git), and typically local. However, it can be synchronized
+with [remote storage](/doc/command-reference/remote) for sharing.
```

Comment:
This I extracted from a previous iteration of the use case, that's why this file was modified. But then I also simplified everything else in this section which is pretty out of scope. Feel free to ignore and I'll move it to it's own PR... Sry

Reply:
OK, reverted these changes for now.